lightning/src/chain/package.rs

```diff
@@ -576,13 +600,22 @@ impl PackageTemplate {
 		}
 		self.height_timer = cmp::min(self.height_timer, merge_from.height_timer);
 	}
-	pub(crate) fn package_amount(&self) -> u64 {
+	/// Gets the amount of all outptus being spent by this package, only valid for malleable
+	/// packages.
+	fn package_amount(&self) -> u64 {
 		let mut amounts = 0;
 		for (_, outp) in self.inputs.iter() {
 			amounts += outp.amount();
 		}
 		amounts
 	}
+	pub(crate) fn package_timelock(&self) -> u32 {
+		let mut max_locktime = 0;
```

Comment:
nit: min_locktime? I would rather call the maximum locktime `std::u32::MAX`

Reply:
Hmm? This is calculating the maximum locktime of all included inputs, so its still a maximum?

Reply:
Ah gotcha, I was thinking about the max absolute value that `nLocktime` might be. `max_default_locktime` would be a bit clearer.

Reply:
I switched to an iterator as Jeff suggested below, so there is no variable name to be confused about :).